spec.html

```diff
@@ -32989,13 +32987,11 @@
 
       <!-- es6num="24.1.2.1" -->
       <emu-clause id="sec-arraybuffer-length">
-        <h1>ArrayBuffer ( _length_ )</h1>
-        <p>ArrayBuffer called with argument _length_ performs the following steps:</p>
+        <h1>ArrayBuffer ( [ _length_ ] )</h1>
```

Comment:
There are a number of legacy cases (most of them constructors or Date methods), this is true, but in the general case we should strive to make the function's length match the number of required arguments.